### PR TITLE
docs: add Flightcontrol as a deployment option

### PR DESCRIPTION
**Pull request by ModupeD**
Added Flightcontrol as a docusaurus deployment option 

## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [x] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [x] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation

Flightcontrol allows users to deploy their Docusaurus project on AWS Fargate directly from their GitHub account.

### CODE REVIEW

File: website/docs/deployment.mdx [hunk 1]
```diff
@@ -819,3 +825,5 @@ See [docs](https://docs.quantcdn.io/docs/cli/continuous-integration) and [blog](
 ## Deploying to Kinsta Application Hosting {#deploying-to-kinsta}
 
 [Kinsta Application Hosting](https://kinsta.com/application-hosting) is a service that lets you build and deploy your web apps directly from your Git repository. Get started in just a minutes by following our [Docusaurus on Kinsta](https://kinsta.com/help/docusaurus-example/) article.
+
+
```
Comment (Josh-Cena) [inline website/docs/deployment.mdx:829]:
```suggestion
```

**Comment by slorber** [comment]
Thanks 👍

**Comment by flybayer** [comment]
Hi @slorber, do you know why the docs website is not yet reflecting this new content?

**Comment by slorber** [comment]
FYI, we are going to clean up our existing Docusaurus docs deployment page in favor of a GitHub discussions category. 

This means your platform won't appear anymore on this page: https://docusaurus.io/docs/deployment

If you want your hosting / CI / CDN platform to remain documented for Docusaurus users, please submit an entry at: https://github.com/facebook/docusaurus/discussions/categories/deployment-platforms

This will be your responsibility to keep it up to date over time.

**Pull request merged**